app_helper_functions.py

```diff
@@ -0,0 +1,65 @@
+import configs
```

Comment:
Love that you did this! We desperately needed to clean up that `app.py` file haha